content/en/llm_observability/ragas_evaluations.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Ragas Evaluations
+further_reading:
+    - link: '/llm_observability/submit_evaluations'
+      tag: 'Documentation'
+      text: 'Submit Evaluations'
+---
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">LLM Observability is not available in the selected site ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+## Overview
+
+[Ragas][1] is an evaluation framework for retrieval augmented generation (RAG) applications. Datadog's Ragas integration enables you to evaluate your production application with faithfulness, answer relevancy, and context precision scores. You can use these scores to find traces that have a high likelihood of inaccurate answers and review them to improve your RAG pipeline.
```

Comment:
```suggestion
[Ragas][1] is an evaluation framework for retrieval augmented generation (RAG) applications. Datadog's Ragas integration enables you to evaluate your production application with scores for faithfulness, answer relevancy, and context precision. You can use these scores to find traces that have a high likelihood of inaccurate answers and review them to improve your RAG pipeline.
```
Just because the original sentence was a garden path for me